File: 자바/콜라츠추측.java
public class 콜라츠추측 {
    public int solution(int num) {
        long number = num;
        for (int i = 0; i < 500; i++) {
            if (number == 1){
                return i;
            }
            number = (number % 2 == 0) ? number/2 : number*3+1;
        }
        return -1;
    }

    public static void main(String[] args) {
        콜라츠추측 qq = new 콜라츠추측();
        System.out.println(qq.solution(626331));
    }
}

File: 콜라츠 추측.py
def solution(num):
    answer = 0
    for i in range(500):
        if num == 1:
            break
        if num % 2 == 0:
            num /= 2
            answer += 1
        else:
            num = num * 3 + 1
            answer += 1

    if num != 1:
        answer = -1

    return answer